src/main/features/claimant-response/views/macro/termsOfAgreementTable.njk

```diff
@@ -4,17 +4,28 @@
       <span class="visually-hidden">{{ t( 'Payment plan' ) }}</span>
     </legend>
 
-    {% if paymentIntention.paymentOption === PaymentType.BY_SPECIFIED_DATE.value %}
+    {% if paymentIntention.paymentOption === PaymentType.IMMEDIATELY.value %}
       {% set amountToPay = 'the full amount' %}
       {% if response.responseType == ResponseType.PART_ADMISSION.value %}
         {% set amountToPay = totalAmount | numeral %}
       {% endif %}
-      {{ row('The offer', t('{{ defendantFullName }} will pay ' + amountToPay + ', no later than {{ paymentDate }}',
+      {{ row('The agreement', t('{{ defendantFullName }} will pay ' + amountToPay + ', immediately.
```

Comment:
[as above](https://github.com/hmcts/cmc-citizen-frontend/pull/1054/files#r229265160)

Reply:
done as part of different story which @karthikkannankainos is working on